src/lib/transition_frontier_controller_tests/stubs.ml

```diff
@@ -544,27 +546,42 @@ struct
       let%bind frontier = Hashtbl.find table peer in
       Sync_handler.transition_catchup ~frontier state_hash
 
-    let get_ancestry {table; logger} peer consensus_state =
+    let mplus ma mb = if Option.is_some ma then ma else mb
+
+    let get_staged_ledger_aux_and_pending_coinbases_at_hash {table; _} peer
```

Comment:
Don't duplicate this code from `coda_lib`. Can you put `get_staged_ledger_aux_and_pending_coinbases_at_hash` in `Sync_handler`. Then, coda_lib and stubs can use the code.